README.md

```diff
@@ -33,11 +33,12 @@ This is a quick installation for use with the cloud environment https://c9.io -
 
 1. If you have a GitHub account, visit https://c9.io and log in with the GitHub button.
 2. Fork this repository to your own GitHub account, creating a `yourname/plots2` project.
-3. Name your project, then enter `yourname/plots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
+3. Name your project, then enter `yournameplots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
 4. In the command line prompt at the bottom of the page, type `. ./install_cloudnine.sh` and press enter.
-5. Enter your username when prompted, and run `rails s -b $IP -p $PORT` when it's done.
-6. You're done! Go to the URL shown!
-
+5. Enter your username when prompted, run bundle install.
+6. To set up the database run 'rake db:setup'.
```

Comment:
I believe @ryzokuken is correct that this step should be run in install_cloudnine.sh, but that script is failing, probably due to lack of a `database.yml` copy step. I'm testing this out right now and will confirm, and add to this PR or open a new one.